checksec

```diff
@@ -289,8 +289,11 @@ chk_file_list() {
     echo_message "RELRO           STACK CANARY      NX            PIE             RPATH      RUNPATH\tSymbols\t\tFORTIFY\tFortified\tFortifiable\tFILE\n" '' '' '{\n'
   fi
 
+  LAST_LINE_NUMBER=$(wc -l < "${CHK_FILE_LIST}")
```

Comment:
This needs to be updated in the src directory, then generate the final code hack/build.sh

Reply:
@slimm609  sorry I missed that. Now the changes are made to the src/functions/chk_file.sh